<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntComWidget</class>
 <widget class="QWidget" name="EntComWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>866</width>
    <height>536</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>866</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EntComWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="FilletWidget" native="true">
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QWidget" name="TopWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>850</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>850</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>850</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 252, 252);
border-bottom:1px solid rgb(222, 222, 222);
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
      </property>
      <widget class="QLabel" name="labelTitle">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>15</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16px &quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
border-bottom:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>10</y>
         <width>32</width>
         <height>28</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#closeButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton#closeButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
